tailwind.config.ts

```diff
@@ -104,6 +104,10 @@ const config: Config = {
         left: "-1px 0 3px 0 rgb(0 0 0 / 0.1), -1px 0 2px -1px rgb(0 0 0 / 0.1)",
         right: "1px 0 3px 0 rgb(0 0 0 / 0.1), 1px 0 2px -1px rgb(0 0 0 / 0.1)",
       },
+      screens: {
+        xs: "512px",
+        xxs: "384px",
+      },
```

Comment:
I'm really hesitant to add new smaller breakpoints since we aren't really going to design for specific breakpoints smaller than tailwinds `sm`. In cases where we need to make tweaks like this, I'd rather use tailwind's arbitrary value support and use something like: `max-[384px]:` responsive prefix.